crypto/err/openssl.txt

```diff
@@ -2695,6 +2698,8 @@ PROV_R_INVALID_SALT_LENGTH:112:invalid salt length
 PROV_R_INVALID_TAG:110:invalid tag
 PROV_R_INVALID_TAGLEN:118:invalid taglen
 PROV_R_MISSING_CEK_ALG:144:missing cek alg
+PROV_R_MISSING_CIPHER:150:missing cipher
```

Comment:
The Travis error is relevant - there is a number conflict.

Reply:
already fixing, thanks

Reply:
I t was due to a rebase and incorrect fixing, unfortunately the update tools do not update the header files when someone change numbers as well, so it is all manual chasing ...